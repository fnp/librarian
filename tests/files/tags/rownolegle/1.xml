<rownolegle>

  <kwestia>
    <akap>Treść</akap>
  </kwestia>

  <blok>
    <kwestia>
      <akap>Treść</akap>
    </kwestia>
    <kwestia>
      <akap>Treść</akap>
    </kwestia>
  </blok>

</rownolegle>
