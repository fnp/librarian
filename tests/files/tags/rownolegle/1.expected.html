<div id="book-text">
<div class="paralell-block paralell-block-first"><div class="kwestia">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">Treść</p>
</div></div>
<div class="paralell-block paralell-block-last">
    <div class="kwestia">
<a href="#f2" class="wl-num">2</a><p class="wl paragraph" id="f2">Treść</p>
</div>
    <div class="kwestia">
<a href="#f3" class="wl-num">3</a><p class="wl paragraph" id="f3">Treść</p>
</div>
  </div>
</div>
